plugins/misc/projects/src/main/java/org/apache/hop/projects/gui/ProjectsGuiPlugin.java

```diff
@@ -239,7 +244,20 @@ public void addNewProject() {
 
         // Save the project-config.json file as well in the project itself
         //
-        project.saveToFile();
+        File configFile = new File(projectConfig.getConfigFilename());
+
+        if (!configFile.exists()) {
```

Comment:
This never works for relative paths. We need to check a path relative to the project location. 
We can use:  `projectConfig.getActualProjectConfigFilename(variables)`

Reply:
I willl fix it. Thanks for the suggestion